src/cli/utils/utils_test.go

```diff
@@ -33,6 +33,52 @@ func (suite *CliUtilsSuite) TestRequireProps() {
 		},
 	}
 	for _, test := range table {
-		test.errCheck(suite.T(), utils.RequireProps(test.props))
+		test.errCheck(suite.T(), RequireProps(test.props))
+	}
+}
+
+func (suite *CliUtilsSuite) TestSplitFoldersIntoContainsAndPrefix() {
+	table := []struct {
+		name    string
+		input   []string
+		expectC []string
+		expectP []string
+	}{
+		{
+			name:    "empty",
+			expectC: selectors.Any(),
+			expectP: nil,
+		},
+		{
+			name:    "only contains",
+			input:   []string{"a", "b", "c"},
+			expectC: []string{"a", "b", "c"},
+			expectP: []string{},
+		},
+		{
+			name:    "only leading slash counts as contains",
+			input:   []string{"a/////", "\\/b", "\\//c\\/"},
+			expectC: []string{"a/////", "\\/b", "\\//c\\/"},
+			expectP: []string{},
+		},
+		{
+			name:    "only prefix",
+			input:   []string{"/a", "/b", "/\\/c"},
+			expectC: []string{},
+			expectP: []string{"/a", "/b", "/\\/c"},
+		},
+		{
+			name:    "mixed",
+			input:   []string{"/a", "b", "/c"},
+			expectC: []string{"b"},
+			expectP: []string{"/a", "/c"},
+		},
+	}
+	for _, test := range table {
+		suite.T().Run(test.name, func(t *testing.T) {
+			c, p := splitFoldersIntoContainsAndPrefix(test.input)
+			assert.Equal(t, test.expectC, c, "contains set")
+			assert.Equal(t, test.expectP, p, "prefix set")
```

Comment:
nit: should probably use `ElementsMatch` as it's not sensitive to order
```suggestion
			assert.ElementsMatch(t, test.expectC, c, "contains set")
			assert.ElementsMatch(t, test.expectP, p, "prefix set")
```